_oasis

```diff
@@ -10,6 +10,8 @@ Plugins:      DevFiles (0.3), META (0.3)
 
 Library stdext
   Path: lib
+  FindlibName: stdext
+  Pack: true
```

Comment:
There's a lot of OASIS autogen-fu in this commit. Can we use this opportunity to switch to `-setup-update dynamic` and reduce this now, and for future commits?